src/main/java/com/example/school/SchoolApplication.java

```diff
@@ -492,11 +493,22 @@ private static List<Activity> findActivitiesForSpecificGroup(List<Activity> acti
     }
 
     //method that get a list of students who was absent in specific day
-    private static Set<UUID> findStudentsAbsentInSpecificDay(List<Activity> activities, OffsetDateTime date) {
+    private static Set<UUID> findStudentsAbsentAtSpecificDate(List<Activity> activities, OffsetDateTime date) {
         return activities.stream()
                 .filter(e -> e.getLesson().getStartDate().toLocalDate().equals(date.toLocalDate()))
                 .filter(e -> !e.isPresent())
                 .map(e -> e.getStudent().getId())
                 .collect(Collectors.toSet());
     }
+
+    //method that get the number of lessons taught by a particular teacher on a particular day
+    private static long findNumberLessonsFromTeacherAtSpecificDay
+                                                (List<Activity> activities, UUID teacherId, OffsetDateTime date) {
```

Comment:
If you want to split line you should do in this way:
`private static long findNumberLessonsFromTeacherAtSpecificDay(List<Lesson> lessons,`
                                                                  `UUID teacherId,`
                                                                  `OffsetDateTime date)`

Reply:
Also it would be better to send in method list of lessons.